docroot/themes/humsci/humsci_basic/docs/lockup-styles.md

```diff
@@ -0,0 +1,29 @@
+# Lockup Styles
+
+| Masthead Theme | Small Screen | Large Screen |
+| -------------- | ------------ | ------------ |
+| Default (light) | ![Default Small Screen] | ![Default Large Screen] |
+| Traditional | ![Default Small Screen Dark Inversion] | ![Default Large Screen Dark Inversion] |
+
+[Default Small Screen]: ./images/lockup-styles/default-small-screen.png
+[Default Large Screen]: ./images/lockup-styles/default-large-screen.png
+[Default Small Screen Dark Inversion]: ./images/lockup-styles/default-small-screen-dark-inversion.png
+[Default Large Screen Dark Inversion]: ./images/lockup-styles/default-large-screen-dark-inversion.png
+
+---
+
+## Anatomy of a Lockup
+<img src="./images/lockup-styles/lockup-anatomy.png" alt="Lockup Anatomy" width="400" />
```

Comment:
I had to hard code these because the image were huge and I needed to set some sort of width. GitHub overrides and sets `max-width: 100%` so if the 400px image is larger than the screen size it'll still scale down.